src/libs/antares/study/area/list.cpp

```diff
@@ -45,6 +46,178 @@ namespace Antares::Data
 {
 namespace // anonymous
 {
+
+enum tristate : uint8_t
+{
+    False = 0,
+    True = 1,
+    Undefined = 2
+};
+
+void readReservesAreaParameters(Antares::Data::Area& area, const IniFile::Section& section)
+{
+    for (auto* p = section.firstProperty; p; p = p->next)
+    {
+        std::string key = p->key;
+        std::ranges::transform(key, key.begin(), [](unsigned char c) { return std::tolower(c); });
+
+        if (key == "energy-activation-ratio-up")
+        {
+            if (!p->value.to<double>(
+                  area.allCapacityReservations.value()
+                    .maxGlobalEnergyActivationRatio[(int)reserve::Direction::UP]))
```

Comment:
i you follow this [comment](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2575/files#r2564490205) you'd simply write: 
```suggestion
                  area.allCapacityReservations.value()
                    .maxGlobalEnergyActivationRatio.up))
```